roles/confluent.control_center/tasks/main.yml

```diff
@@ -68,6 +68,15 @@
     mode: '764'
     recurse: yes
 
+- name: Create RocksDB Directory
+  file:
+    path: "{{control_center_rocksdb_path}}"
+    group: "{{control_center.group}}"
+    owner: "{{control_center.user}}"
+    mode: '764'
+    recurse: yes
```

Comment:
Do we need the recurse? I've been looking into idempotency when rerunning the playbooks and the recurse flag seems to make the rerun show "changed" rather than "skipped"

Reply:
It may be possible, but would require the user to create the parent directories in advance in either a pre-tasks block or by hand. But would limit their ability to control parent dir permissions to the scope of the control-center user/group. As that user/group wouldn't exist yet in a new install.

Reply:
I assumed you meant the other way around with those tasks, recursion create first permissions second. But this is made.